src/views/Home.vue

```diff
@@ -84,6 +91,67 @@ export default {
 		}
 	},
 	methods: {
+		moveMessage(e) {
+
+			// We don't want to use this.activeAccount here because we want it to work in the unified inbox
+			var startAccount = e.items[0].attributes.href.baseURI.match(/message\/(\d)/)
+			var targetAccount = e.droptarget.href.match(/accounts\/(\d)/)
+
+			if (targetAccount[1] == startAccount[1]) {
```

Comment:
general coding tip: use *early return*s for the unlikely condition. Then the rest of the function doesn't have to be indented and the structure is a lot easier to read.